util/typeutil/stringInt.go

```diff
@@ -0,0 +1,32 @@
+package typeutil
+
+import (
+	"encoding/json"
+	"strconv"
+)
+
+type StringInt int
```

Comment:
There's probably a better name we could use, though I don't have an elegant suggestion. Perhaps other reviewers might have an idea.

Reply:
The another name that came up is StringOrInt but I find StringInt more concise.